src/blueapi/service/facade.py

```diff
@@ -0,0 +1,73 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+from blueapi.service.model import DeviceModel, PlanModel, WorkerTask
+from blueapi.worker.event import WorkerState
+from blueapi.worker.task import RunPlan
+from blueapi.worker.worker import TrackableTask
+
+
+class BlueskyContextFacade(ABC):
```

Comment:
I'm not sure about this name now either as the methods here are related to both the `BlueskyContext` and the `Worker`.

Reply:
Have changed name.